File: 01gpio/03digital_in/interrupt/interrupt_wpi.py
import wiringpi as wpi
import time

PIN_SW = 18
wpi.wiringPiSetupGpio()
wpi.pinMode(PIN_SW, wpi.INPUT)

def isr():
	print(wpi.digitalRead(PIN_SW))

wpi.wiringPiISR(PIN_SW, wpi.INT_EDGE_BOTH, isr);

while True:
	time.sleep(100)

File: 01gpio/03digital_in/interrupt/interrupt_wpi.js
var wpi = require('node-wiring-pi');
var pin = 23;

wpi.setup('gpio');
wpi.pinMode(pin, wpi.INPUT);

wpi.wiringPiISR(pin, wpi.INT_EDGE_BOTH, function(){
	console.log(wpi.digitalRead(pin));
});


